resources/scripts/jquery/jquery-utils.js

```diff
@@ -308,6 +308,30 @@
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
+		var name = document.getElementsByName('name')[0].value;
+           	var abbrev = document.getElementsByName('abbrev')[0].value;
+           	var title = document.getElementsByName('title')[0].value;
+
+                if (name.startsWith('urn:') || name.startsWith('file:')) {
+                    alert("The name cannot be in a urn:// or a file:// protocol.")
+                    return false;} else {
+                       try {
+                           nameURL = new URL(name);
+                           } catch (_) {
+                           return alert("Enter a valid URI as name.");
+                        }
+                    }
+
+                if (abbrev == "") {
```

Comment:
I would move the retrieval for `abbrev` here. Since this code will not run, If the name check failed. And it makes it more obvious which parts of the code belong together.

Reply:
True.